Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java

```diff
@@ -213,7 +213,7 @@ void verifyDateLabelColorAndToolTip(CalendarPanel panel, int labelIdx, Color bgC
     public void TestYearEditor() throws NoSuchFieldException, IllegalAccessException, NoSuchMethodException, InvocationTargetException
     {
         verifyYearEditor(YearEditorFinalizer.doneEditingButton);
-        //verifyYearEditor(YearEditorFinalizer.yearTextField); uncomment once code for issue #113 has been implemented
+        verifyYearEditor(YearEditorFinalizer.yearTextField);
```

Comment:
I uncommented this test for the new case *finalize year editing with enter key* on your behalf. I hope you don't mind.